README.md

```diff
@@ -314,6 +316,16 @@ Other optional arguments to `nlsolve`, available for all algorithms, are:
 * `extended_trace`: should additifonal algorithm internals be added to the state
   trace? Default: `false`.
 
+## Fixed Points 
+
+There is a `fixedpoint()` wrapper around `nlsolve()` which maps an input function `F(x)` to `G(x) = F(x) - x`, and likewise for the in-place. This allows convenient solution of fixed-point problems, e.g. of the kind commonly encountered in computational economics. Some notes:
+
+* The default method is `:anderson` with `m = 0`, which corresponds to simple iteration on a contraction mapping.
```

Comment:
It's clear that this will work on a contraction, but you could put in something that's not a contraction mapping, and hope for the best (not saying you should, but this is done in some cases).

Maybe something like: "The default method is `:anderson` with `m = 0`. This algorithm is robust when `F(x)` is a contraction mapping, but may not be very efficient when the Lipschitz constant is close to 1."

Reply:
Right, you could do that and add that caveat. I've no issues with it. I just wanted to keep the docs as short as possible. But will amend the PR to make this clear.

Reply:
Done.